docs/pages/x/api/date-pickers/date-calendar.json

```diff
@@ -14,19 +15,25 @@
     "disablePast": { "type": { "name": "bool" } },
     "displayWeekNumber": { "type": { "name": "bool" } },
     "fixedWeekNumber": { "type": { "name": "number" }, "default": "undefined" },
+    "focusedView": {
+      "type": {
+        "name": "enum",
+        "description": "'day'<br>&#124;&nbsp;'month'<br>&#124;&nbsp;'year'"
+      }
+    },
     "loading": { "type": { "name": "bool" } },
     "maxDate": { "type": { "name": "any" } },
     "minDate": { "type": { "name": "any" } },
     "onChange": { "type": { "name": "func" } },
+    "onFocusedViewChange": { "type": { "name": "func" } },
     "onMonthChange": { "type": { "name": "func" } },
     "onViewChange": { "type": { "name": "func" } },
     "onYearChange": { "type": { "name": "func" } },
     "openTo": {
       "type": {
         "name": "enum",
         "description": "'day'<br>&#124;&nbsp;'month'<br>&#124;&nbsp;'year'"
-      },
-      "default": "'day'"
```

Comment:
We already lost this precision on the new pickers due to the shared interface.

It would be nice to be able to do something like:

```tsx
/**
 * @propDefaultValue view "day"
**/
interface MyComponentProps extends SomeShareInterface {}
```

Not sure about the exact synthax
This would allow to define doc default values without redefining the type itself.

If you know about any existing convention for this use case I'm very interested

Reply:
This specific use case does not seem to be supported by jsdocs.

You have the list of "supported tags" [here](https://jsdoc.app/index.html).

We could add a plugin to support an additional tag `@overrideDefault`. But it sounds to be an interesting project, but will require some time to navigate int jsdocs documentation